pkg/nfd-master/nfd-master.go

```diff
@@ -262,6 +273,40 @@ func (m *nfdMaster) Run() error {
 	}
 }
 
+func (m *nfdMaster) handleLabelerError(err error, nodeName string) {
```

Comment:
nit: looks a bit strange that this is a separate method 🧐 How about just doing this error checking inline in `processNodeLabelRequest`